rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/release/ReleaseController.java

```diff
@@ -584,6 +610,70 @@ public ResponseEntity<EntityModel<Release>> createRelease(
         return ResponseEntity.created(location).body(halResource);
     }
 
+    @PreAuthorize("hasAuthority('WRITE')")
+    @PatchMapping(value = RELEASES_URL + "/{id}/spdx")
+    public ResponseEntity<?> updateSPDX( @RequestBody Map<String, Object> reqBodyMap, @PathVariable("id") String releaseId) throws TException {
```

Comment:
Please document the endpoint with `@Operation` annotation for OpenAPI docs.

Reply:
And the parameters as well with `@Parameter` annotation.

Reply:
Added `@Operation` and `@Parameter`